modules/virt-pausing-vm-web.adoc

```diff
@@ -19,15 +19,15 @@ You can pause a virtual machine (VM) from the web console.
 * To stay on this page, where you can perform actions on multiple VMs:
 
 .. Click the Options menu {kebab} located at the far right end of the row and click *Pause VirtualMachine*.
+.. If action confirmation is enabled, in the confirmation dialog, click *Pause*.
```

Comment:
I wonder if *Pause* should go first, as it is more important than _confirmation dialog_.

```suggestion
.. If action confirmation is enabled, click *Pause* in the confirmation dialog.
```